roles/ceph-osd/tasks/scenarios/collocated.yml

```diff
@@ -26,6 +28,10 @@
     - not osd_auto_discovery
     - containerized_deployment
     - item.0.partitions|length == 0
+  register: osd
```

Comment:
Calling the `register` the same name won't work since ansible will overwrite the previous one, even if the task is skipped. So you have to use different names for each task.

Reply:
That's the point.  I want it to overwrite each one because the real state variable is osd_created

Reply:
Yes, in this approach there is no issue to register multiple times the same variable name.
It might be an issue if we wanted to have the possibility to deploy multiple osd scenario per host though.

Reply:
@guits Is that supported today?  Are you asking me to change it or this is OK?

Reply:
@trozet this is OK, I was just trying to clarify.